theme/common.less

```diff
@@ -716,6 +716,12 @@ Avatar
     width: 0;
 }
 
+/* Modulator */
```

Comment:
UI elements for the modulator are defined in pxt. and we should hide them for all targets that don't need them.